Formula/flow.rb

```diff
@@ -20,10 +20,6 @@ class Flow < Formula
   uses_from_macos "rsync" => :build
   uses_from_macos "unzip" => :build
 
-  on_linux do
-    depends_on "elfutils"
```

Comment:
@iMichka Note that this will need removed because it doesn't exist in homebrew-core.

Reply:
No issue, it was added by mistake.

I need to either port these formulae to mac, or start adding them as Linux-only: but in this case I will also add minimal Linux-Ci to make sure it is tested. I’ll start on working on this in September

Reply:
Cool, thanks. Don't worry about CI, it'll be caught in future by the new CI as-of this PR.